shell.nix

```diff
@@ -24,6 +24,6 @@
   venvDir = "./.venv";
```

Comment:
I wonder if this needs the same fix btw, otherwise the `.venv` folder might end up under `relayer/`

Reply:
tried out the change but had to undo it since the CI immediately failed 🤷🏼